flask_socketio/__init__.py

```diff
@@ -473,7 +473,11 @@ def run(self, app, host=None, port=None, **kwargs):
             def run_server():
                 import eventlet
                 import eventlet.wsgi
-                eventlet_socket = eventlet.listen((host, port))
+                import socket
+                addresses = socket.getaddrinfo(host, port)
```

Comment:
Any reason to use Python's standard socket here, instead of eventlet's own functions, which are green friendly?